Tests/BijectiveDictionaryTests/BijectiveDictionaryTests.swift

```diff
@@ -299,4 +299,92 @@ func conflict() {
     #expect(dict.conflict(with: ("A", 1)) == .pair)
     #expect(dict.conflict(with: ("A", 3)) == .both(otherLeft: "C", otherRight: 1))
 }
+
+@Test
+func uniquingWithNoConflicts() {
+    let pairs = [("A", 1), ("B", 2), ("C", 3)]
+    var conflicts = [BijectiveDictionary<String, Int>.Conflict]()
+    
+    let dict = BijectiveDictionary(pairs) { pair, conflict  in
```

Comment:
Because we're using a trailing closure here, the second parameter is not visible here. It's not clear what the purpose of the trailing closure is here. 

Perhaps the first parameter should have a required label at the call site to make this more clear. 

Then again, perhaps this is nitpicky and unimportant. 
IdentifiedCollections has a nearly identical initializer, and they don't seem to mind. 

https://swiftpackageindex.com/pointfreeco/swift-identified-collections/main/documentation/identifiedcollections/identifiedarray/init(_:uniquingidswith:)